### PR TITLE
Add casting remarks to EventLogInformation.Attributes (#1126)

### LINKED ISSUE #1126: Add this to the System.Diagnostics.Eventing.Reader.EventLogInformation.Attributes documentation
Please consider adding the following to the System.Diagnostics.Eventing.Reader.EventLogInformation.Attributes documentation.

```
System.Diagnostics.Eventing.Reader.EventLogInformation eli;

System.IO.File​Attributes fa = (System.IO.File​Attributes)Enum.ToObject(typeof(System.IO.File​Attributes), eli.Attributes.Value);
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ac61b9c6-24cb-eb42-b879-639b81575d26
* Version Independent ID: c1031585-2f4a-be0e-1751-6d4487680508
* Content: [EventLogInformation.Attributes Property (System.Diagnostics.Eventing.Reader)](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.eventing.reader.eventloginformation.attributes?view=netframework-4.7.2)
* Content Source: [xml/System.Diagnostics.Eventing.Reader/EventLogInformation.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Diagnostics.Eventing.Reader/EventLogInformation.xml)
* Product: **dotnet-api**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

### PR DESCRIPTION
**Pull request by JJS**
## Summary

Added a remark section containing the proposed text and referencing a cs and vb snippet.

Fixes #1126

Depends on https://github.com/dotnet/samples/pull/1649

### CODE REVIEW

File: xml/System.Diagnostics.Eventing.Reader/EventLogInformation.xml [hunk 1]
```diff
@@ -60,7 +60,15 @@
       <Docs>
         <summary>Gets the file attributes of the log file associated with the log.</summary>
         <value>Returns an integer value. This value can be null.</value>
-        <remarks>To be added.</remarks>
+      <format type="text/markdown"><![CDATA[
```
Comment (mairaw) [inline xml/System.Diagnostics.Eventing.Reader/EventLogInformation.xml]:
```suggestion
        <remarks>
          <format type="text/markdown"><![CDATA[
```

File: xml/System.Diagnostics.Eventing.Reader/EventLogInformation.xml [hunk 2]
```diff
@@ -60,7 +60,15 @@
...
-        <remarks>To be added.</remarks>
+      <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The values of this property are the same as those of the <xref:System.IO.FileAttributes> type. To make working with this property easier, you can convert the value to that type using the following code.  
+  
+ [!code-csharp[System.Diagnostics.Eventing.Reader.EventLogInformation#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/System.Diagnostics.Eventing.Reader.EventLogInformation/cs/attributes.cs#1)]
+ [!code-vb[System.Diagnostics.Eventing.Reader.EventLogInformation#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/System.Diagnostics.Eventing.Reader.EventLogInformation/vb/attributes.vb#1)] 
+
+ ]]></format>
```
Comment (mairaw) [inline xml/System.Diagnostics.Eventing.Reader/EventLogInformation.xml]:
```suggestion
 ]]></format>
        </remarks>
```

File: xml/System.Diagnostics.Eventing.Reader/EventLogInformation.xml [hunk 3]
```diff
@@ -60,7 +60,15 @@
       <Docs>
         <summary>Gets the file attributes of the log file associated with the log.</summary>
         <value>Returns an integer value. This value can be null.</value>
-        <remarks>To be added.</remarks>
+      <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The values of this property are the same as those of the <xref:System.IO.FileAttributes> type. To make working with this property easier, you can convert the value to that type using the following code.
```
Comment (mairaw) [inline xml/System.Diagnostics.Eventing.Reader/EventLogInformation.xml]:
```suggestion

The values of this property are the same as those of the <xref:System.IO.FileAttributes> type. To make working with this property easier, you can convert the value to that type using the following code:
```

**Comment by JJS** [comment]
I totally missed the remarks tag and thus the indentation went wrong, sorry for that.

**Comment by mairaw** [comment]
Rebuilding with the merged samples.

**Comment by mairaw** [comment]
Trying again since we were still getting warnings.

**Pull request merged**